src/main/drivers/serial_uart_pinconfig.c

```diff
@@ -46,6 +46,12 @@ void uartPinConfigure(const serialPinConfig_t *pSerialPinConfig)
     for (size_t hindex = 0; hindex < UARTDEV_COUNT_MAX; hindex++) {
```

Comment:
For full configurability, `uartHardware` now has info about all possible UARTs, and entry count is `UARTDEV_COUNT_MAX`.

`UARTDEV_COUNT` is number of USE_UARTx defined, which is different from `UARTDEV_COUNT`.

`UARTDEV_COUNT` will become `UARTDEV_COUNT_MAX` when fully configurable target.h is actually written, but they are different by nature.